src/DF950.c

```diff
@@ -91,21 +273,164 @@ s32 func_80149828(void) {
     return D_80159AD4;
 }
 
-INCLUDE_ASM(s32, "DF950", sfx_stop_env_sounds);
+void sfx_stop_env_sounds(void) {
+    s32 i;
```

Comment:
Same notes here basically about the one above. You may be able to initialize sound on line 1 like the prev function too

Reply:
sound can't be initialized on line 1 because D_80159AD0 gets assigned to in the first if statement.

Reply:
Oh I'm dumb. It gets incremented in the loop. okay disregard 😁